content/test/manual/service/reporters/SQSDeploymentReporter.spec.ts

```diff
@@ -0,0 +1,26 @@
+import { Entity, EntityType } from "@katalyst/content/service/Entity";
+import { SQSDeploymentReporter } from "@katalyst/content/service/reporters/SQSDeploymentReporter";
+import { EnvironmentConfig, Environment } from "@katalyst/content/Environment";
+
+describe("SQS Deployment Reporter", () => {
+
+    fit(`Simple event reporting`, async () => {
```

Comment:
You should probably change to `it`

Reply:
It doens't run on Circle CI. I think that's the reason this is commented for now :(